src/components/Comments/Comment.js

```diff
@@ -242,28 +277,58 @@ const Comment = ({
         {isLast && level >= 2 ? (
           <View testID="level2" style={styles.gap} />
         ) : (
-          <TouchableOpacity activeOpacity={1} onPress={onPress} testID="memoComment">
-            <ButtonHightlight
-              onLongPress={handleOnLongPress}
-              style={styles.btnReply}
-              onPress={onPress}>
-              <MemoCommentReply />
-              <Text style={styles.btnReplyText}>Reply</Text>
-            </ButtonHightlight>
-          </TouchableOpacity>
+          <>
+            <TouchableOpacity activeOpacity={1} onPress={onPress} testID="memoComment">
+              <ButtonHightlight
+                onLongPress={handleOnLongPress}
+                style={[
+                  styles.btnReply,
+                  {
+                    marginRight: 0,
+                    paddingRight: 0
+                  }
+                ]}
+                onPress={onPress}>
+                <MemoCommentReply />
+              </ButtonHightlight>
+            </TouchableOpacity>
+            {!comment.is_you && (
+              <TouchableOpacity onPress={onPressDm} testID="sendDMbtn" activeOpacity={1}>
+                <ButtonHightlight
+                  onPress={onPressDm}
+                  style={[
+                    styles.btnBlock(comment.user.id === yourselfId),
+                    styles.btn,
+                    {
+                      marginRight: 0,
+                      paddingRight: 0
+                    }
+                  ]}>
+                  <MemoSendDM />
+                </ButtonHightlight>
+              </TouchableOpacity>
+            )}
+          </>
         )}
 
         <ButtonHightlight
           onLongPress={handleOnLongPress}
-          style={[styles.btnBlock(comment.user.id === yourselfId), styles.btn]}
+          style={[
+            styles.btnBlock(comment.user.id === yourselfId),
+            styles.btn,
+            {
+              marginRight: 0,
+              paddingRight: 0
+            }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The structure of the comment component has been significantly changed with the addition of the direct message (DM) button. Ensure that these changes don't affect the layout and functionality of the component in unexpected ways.



<!-- This is an auto-generated comment by OSS CodeRabbit -->